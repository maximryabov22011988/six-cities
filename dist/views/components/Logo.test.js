"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const React = require("react");
const renderer = require("react-test-renderer");
const react_router_dom_1 = require("react-router-dom");
const Logo_1 = require("./Logo");
it('Logo renders correctly with mandatory props', () => {
    const tree = renderer
        .create(React.createElement(react_router_dom_1.BrowserRouter, null,
        React.createElement(Logo_1.default, { position: "header" }))).toJSON();
    expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Logo.test.js.map