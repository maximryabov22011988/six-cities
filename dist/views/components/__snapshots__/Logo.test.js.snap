// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly with mandatory props 1`] = `
<a
  className="header__logo-link"
  href="/"
  onClick={[Function]}
>
  <img
    alt="6 cities logo"
    className="header__logo"
    height="41"
    src="img/logo.svg"
    width="81"
  />
</a>
`;
